docs/UserGuide.md

```diff
@@ -102,20 +102,28 @@ This project is built on top of [AddressBook-Level3 Project](https://se-educatio
+| [**rview**](#listing-rental-information-of-a-client-rview) | View rental information of a client     | `rview CLIENT_INDEX`                                                                                                                                           | `rview 1`                                                                    |
+| [**cedit**](#editing-a-client-cedit)                       | Edit a client                           | `cedit CLIENT_INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL]`                                                                                                       | `cedit 1 n/Steven Tan e/steventan@abc.com p/98765432`                        |
+| [**redit**](#editing-a-client-s-rental-information-redit)  | Edit rental information for a client    | `redit c/CLIENT_INDEX r/RENTAL_INDEX [a/ADDRESS] [s/RENTAL_START_DATE] [e/RENTAL_END_DATE] [dd/RENT_DUE_DATE] [m/MONTHLY_RENT] [d/DEPOSIT] [cl/CUSTOMER_LIST]` | `redit c/1 r/1 a/65 Anderson Road m/3000 d/0`                                |
+| [**find**](#locating-clients-find)                         | Find information                        | `find [k/KEYWORDS...] [n/NAME...] [p/PHONE_NUMBER...] [e/EMAIL...] [t/TAG]...`                                                                                 | `find k/Tan n/Steven Tan p/98765432`                                         |
+| [**cdelete**](#deleting-a-client-cdelete)                  | Delete a client                         | `cdelete CLIENT_INDEX`                                                                                                                                         | `cdelete 3`                                                                  |
+| [**rdelete**](#deleting-a-rental-information-rdelete)      | Delete rental information from a client | `rdelete c/CLIENT_INDEX r/RENTAL_INDEX`                                                                                                                        | `rdelete c/1 r/2`                                                            |
+| [**sort**](#sorting-all-entries-sort)                      | Sort clients by name                    | `sort`                                                                                                                                                         | -                                                                            |
+| [**↑up ↓down arrow key**](#sorting-all-entries-sort)       | Navigating command history              | `↑ up-arrow key` and `↓ down-arrow key`                                                                                                                        | -                                                                            |
```

Comment:
Should 'navigating' be 'navigate' to use imperative tone?

Reply:
Okay thanks